src/components/book.vue

```diff
@@ -1,118 +1,122 @@
-                    <div v-for="(book, key) in result.similar_books" v-bind:key="key">
-                        <v-list-tile :key="book.title" avatar>
-                      <router-link :to="'/book/Goodreads/'+book.id">
-                         <v-list-tile-avatar>
-                        <img :src="book.small_image_url" >
-                      </v-list-tile-avatar>
-                      </router-link>
-                      <v-list-tile-content>
-                      <router-link :to="'/book/Goodreads/'+book.id">
-                        <v-list-tile-title v-html="book.title"></v-list-tile-title>
-                      </router-link>
-                        <v-list-tile-sub-title><span class='text--primary'>{{book.authors.author.name}}</span>&mdash; {{book.average_rating}}</v-list-tile-sub-title>
-                      </v-list-tile-content>
-                    </v-list-tile>
-                    </div>
-                </div>
+                                            <div v-for="(author, index) in result.authors" :key="index">
+                                                <v-list-tile :key="author.name" >
+                                                    <v-list-tile-content>
+                                                        <a :href=author.link target="_blank">
+                                                            <v-list-tile-title v-html="author.name"></v-list-tile-title>
+                                                        </a>         <v-list-tile-sub-title><span class='text--primary'>{{author.average_rating}}</span></v-list-tile-sub-title>
+                                                    </v-list-tile-content>
+                                                </v-list-tile>
+                                            </div>
+                                        </span>        <span>
+                                            <v-subheader>Similar Books</v-subheader>
+                                            <div>
+                                                <li v-for="book in similar">
+                                                    <router-link :to="'/search/'+book.Name">{{book["Name"]}}</router-link>
+                                                </li>
 
-        </span>
-          </div>
-</v-card-title>
-<!-- @TODO FEATURES TO BE ADDED-->
-        <v-card-actions>
-          <v-btn flat color="orange">read</v-btn>
-          <v-btn flat color="orange">share</v-btn>
-</v-card-actions>
-                </div>
-                                    <div v-else-if="result.error">  
-                      <v-list-tile>
-                      <v-list-tile-content>
-                        <v-list-tile-title >No search results found in {{index}}.</v-list-tile-title>
-                      </v-list-tile-content>
-                    </v-list-tile>
-                      </div>
-                    <v-divider ></v-divider>
+                                                {{similarBooks}}
```

Comment:
what does this do ?

Reply:
This is a temporary hack to refresh the value of similarBooks since it contains a Promise and it needs to be notified when the Promise is resolved.